### PR TITLE
docs: adding rough contributing guidelines

### PR DESCRIPTION
**Pull request by aidanjgriffiths**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - [Dry vs Wet](https://medium.com/@nrk25693/dry-or-wet-and-why-867ac3096483)
+  - [What is DRY code?](https://codinglead.github.io/javascript/what-is-DRY-code)
+  - [Simplicity Over Complexity](https://en.wikipedia.org/wiki/Zen_of_Python)
+  - [Readability Over optimisation](https://en.wikiquote.org/wiki/Donald_Knuth#Computer_Programming_as_an_Art_(1974))
+- Pull Requests
+  - [GitHub Flow](https://guides.github.com/introduction/flow/index.html)
+  - [Strive for Schemantic/Conventional Commits](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716)
+  - [Code Reviews](https://google.github.io/eng-practices/review/reviewer/)
+    - Each Pull Request must have merging approval of at least one (ideally multiple) senior member before merging to main branch.
+    -
```
Comment (namvnn) [inline .github/CONTRIBUTING.md]: Remove this line since it has no information.

**Comment by aidanjgriffiths** [comment]
@nnfunny I'm finished with add a bit more to the documentation.
Let me know if its ok to be merged.

Thanks.

**Comment by namvnn** [comment]
@aidanjgriffiths Your changes to the documentation are good to be merged. You can merge when the Github action finishes.

**Comment by aidanjgriffiths** [comment]
Since I've changed the `development` branch to the default I will close this PR and merge to the `development` branch.

**Pull request closed without merging**